ui/App/ProjectScreen/Source/Commit.svelte

```diff
@@ -153,6 +192,38 @@
       </div>
     </div>
 
+    {#each anchors as anchor}
+      <div class="anchor">
+        <div class="header">
```

Comment:
Can we call the class `anchor-header` to make disambiguate it from other headers in this component?

Reply:
aa9bc1f7422719b8afef06ffe2faeb48cffa0f75